example/packet/packet_dump_run.sh

```diff
@@ -6,15 +6,23 @@
 # SPDX-License-Identifier:     BSD-3-Clause
 #
 
-PCAP_IN=`find . ${TEST_DIR} $(dirname $0) -name udp64.pcap -print -quit`
+if  [ -f ./interface_env ]; then
+	. ./interface_env
+else
+        echo "BUG: unable to find interface_env!"
+        echo "interface_env has to be in current directory"
+        exit 1
+fi
 
-echo "Packet dump test using PCAP_IN = ${PCAP_IN}"
+setup_interfaces
 
-./odp_packet_dump${EXEEXT} -i pcap:in=${PCAP_IN}:loops=10 -n 10 -o 0 -l 64
+./odp_packet_dump${EXEEXT} -i $IF0 -n 10 -o 0 -l 64
 STATUS=$?
 if [ "$STATUS" -ne 0 ]; then
   echo "Error: status was: $STATUS, expected 0"
   exit 1
 fi
 
+cleanup_interfaces
+
```

Comment:
In addition to this one there is `packet/pktio_run.sh` script.

Reply:
Ack.